Resources/Azure/ArmTemplates/template.json

```diff
@@ -277,7 +277,9 @@
   "variables": {
     "deploymentUrlBase": "https://raw.githubusercontent.com/SkillsFundingAgency/dfc-devops/master/ArmTemplates/",
     "appInsightsName": "[concat(parameters('projectPrefix'),'-ai')]",
```

Comment:
Currently this will create: dfc-foo-bar-ai (app insights for web app) and dfc-foo-bar-fa-ai (app insights for fn app)

For consistency (we've done this on the other apps) you probably want to change this to 
"appInsightsName": "[concat(variables('appServiceName'),'-ai')]",

This change will create: dfc-foo-bar-as-ai (app insights for web app) and dfc-foo-bar-fa-ai (app insights for fn app)

Reply:
updated app service appInsightsName